corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -65,6 +73,20 @@
     private final CorfuRuntime runtime;
     private final String diskPath;
 
+    private final ConcurrentMap<String, DescriptorProtos.FileDescriptorProto>
+        fdProtoMap = new ConcurrentHashMap<>();
+    /*
+     * Maps the Message name to the name of the FileDescriptorProto containing it.
+     */
+    private final ConcurrentMap<String, String> messagesFdProtoNameMap =
+        new ConcurrentHashMap<>();
+
+    /**
+     * This is used as a file descriptor cache. Used for optimization.
+     */
+    private final ConcurrentMap<String, Descriptors.FileDescriptor>
```

Comment:
This is also present in the DynamicProtobufSerializer class, perhaps you just need a `@Getter` method on that right?

Reply:
got it